lavalink/playermanager.py

```diff
@@ -2,6 +2,10 @@
 from .models import BasePlayer
 
 
+class NodesNotFound(Exception):
+    """The exception will be raised when their are no nodes found to connect to."""
```

Comment:
... when **there** are no nodes available.

Reply:
Perhaps this can be changed to a generic `NodeException`? I don't mind either way but a generic exception would allow it to be used in more places. Also perhaps this can be moved to an `exceptions.py` file, I wouldn't say this belongs in `playermanager.py`.